src/pin.rs

```diff
@@ -0,0 +1,219 @@
+//! Mock digital InputPin and OutputPin implementations
+//! 
+//! ```
+//! use std::io::ErrorKind;
+//! 
+//! use embedded_hal_mock::MockError;
+//! use embedded_hal_mock::pin::{Transaction as PinTransaction, Mock as MockPin};
```

Comment:
Nitpicking: In other docs we always used the target type as prefix: `I2cMock` and `I2cTransaction`, `SpiMock` and `SpiTransaction`. Here we use `PinTransaction` but `MockPin`.